2_delivery_care.do

```diff
@@ -23,6 +23,12 @@ gen country = regexs(1) if regexm(country_year, "([a-zA-Z]+)")
 	replace `var' = . if !inlist(`var',0,1)
 	
 	 }
+	 if inlist(name, "Senegal2019") {

```

Comment:
I am wondering why this is changed back to the older version instead of the latest one (the latest version in master), which I have agreed to your changes on Senegal2018 and merged it to master. I suspect it's the fetch and rebase practice is not done when you were modifying the template for Senegal2019?

Reply:
Hi Aline, I actually did the fetch, and then rebase from the master before opening these files. There were several conflicts after rebasing, so I adopted the changes from the master branch manually. I don't know why it changes back to the older version. 

I have just rebased and then changed the codes again for re-push.